### PR TITLE
Upgrade to 2.12.8

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

File: tests/jvm/src/test/resources/metac-metacp.diff [hunk 1]
```diff
@@ -14,10 +14,9 @@ advanced/D#
                  kind: TYPE
                  properties: 4
                  display_name: "_"
-     self {
```
Comment (olafurpg) [inline tests/jvm/src/test/resources/metac-metacp.diff]: I'm not sure what's happening there 🤔

File: tests/jvm/src/test/resources/metac-metacp.diff [hunk 2]
```diff
@@ -14,10 +14,9 @@ advanced/D#
...
      declarations {
-+      symlinks: "advanced/D#_$2#"
```
Comment (olafurpg) [inline tests/jvm/src/test/resources/metac-metacp.diff]: Considering this is a generated symbol and we don't sort the declarations I'm willing to accept this change.

File: tests/jvm/src/test/resources/metac-metacp.diff [hunk 3]
```diff
@@ -329,25 +328,6 @@ example/Synthetic#x$2.
 +  private_this_access {
 
 
-================
```
Comment (olafurpg) [inline tests/jvm/src/test/resources/metac-metacp.diff]: LGTM 👍

**Comment by olafurpg** [comment]
I have NB published semanticdb-scalac v4.1.0 for 2.12.8

File: tests/jvm/src/test/resources/metac-metacp.diff (hunk 1 above)
Comment (xeno-by) [reply tests/jvm/src/test/resources/metac-metacp.diff]: This isn't a diff introduced by the PR though.

File: tests/jvm/src/test/resources/metac-metacp.diff (hunk 2 above)
Comment (xeno-by) [reply tests/jvm/src/test/resources/metac-metacp.diff]: Likewise.

File: tests/jvm/src/test/resources/metac-metacp.diff (hunk 3 above)
Comment (xeno-by) [reply tests/jvm/src/test/resources/metac-metacp.diff]: Neato.

**Comment by xeno-by** [comment]
Looks like Travis is unhappy. Other than that, lgtm.

**Pull request merged**